src/ngraph/frontend/onnx_import/op/non_zero.cpp

```diff
@@ -0,0 +1,42 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <memory>
+
+#include "ngraph/opsets/opset2.hpp"
+#include "non_zero.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                NodeVector non_zero(const Node& node)
+                {
+                    const Output<ngraph::Node> data = node.get_ng_inputs().at(0);
```

Comment:
This isn't required but I'd suggest changing `Output<ngraph::Node>` to `auto`. The `data` type will then be deduced to `shared_ptr<Node>` and this will be forwarded to the `NonZero` constructor. This will be more future-proof if for any reason the way ops are connected in the graph changes.